deployments/cloudfoundry/index.yml

```diff
@@ -18,3 +18,4 @@
 1.28.0: https://github.com/signalfx/splunk-otel-java/releases/download/v1.28.0/splunk-otel-javaagent.jar
 1.29.0: https://github.com/signalfx/splunk-otel-java/releases/download/v1.29.0/splunk-otel-javaagent.jar
 1.30.0: https://github.com/signalfx/splunk-otel-java/releases/download/v1.30.0/splunk-otel-javaagent.jar
+2.0.0-alpha: https://github.com/signalfx/splunk-otel-java/releases/download/v2.0.0-alpha/splunk-otel-javaagent.jar
```

Comment:
```suggestion
```

Prefer not to mess with CF while in alpha phase here.

Reply:
the 2.0.0-alpha version is also used in https://github.com/signalfx/splunk-otel-java/blob/main/deployments/cloudfoundry/buildpack/README.md#configuration might be easier to let users decide which version they want

Reply:
Bah ok, I think you're right. I honestly am not sure if cloudfoundary (or at least the way our buildpack works) is compatible with multiple branches/major releases, but it does probably make sense to keep all the versions on main. I'll add it back in another PR that also adds the 1.30.1 patch.